inits.json

```diff
@@ -0,0 +1,54 @@
+{
+    "user_info": [
+        {
+            "Directory with FITS files": "./sample-data/HatP32Dec202017",
+            "Directory to Save Plots": "./sample-data",
+            "Directory of Flats": "n",
+            "Directory of Darks": "n",
+            "Directory of Biases": "n",
+
+            "AAVSO Output?": "y",
+            "AAVSO Observer Account Number": "RTZ",
+            "Secondary Observer Codes": "n",
+
+            "Observation date": "December 17, 2017",
+            "Obs. Latitude (+=N,-=S)": "+31.68",
```

Comment:
We might want to put in a comment the format of the Lat and Long for the user.

Reply:
Left in a comment for how to format latitude and longitude.